bin/delete_anonymised

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
 LAA_REF=$1
+DESTINATION_ENVIRONMENT=$2
 
-bundle exec rails anonymised:delete\[$LAA_REF\]
+if [[ $DESTINATION_ENVIRONMENT == "local" ]]; then
+  bundle exec rails anonymised:delete\[$LAA_REF\]
```

Comment:
So we can just do `if != local` here and put the `kubectl` command in that as both conditions run the same line

Reply:
Ah but the kubectl command and the `bundle exec rails` aren't separate commands, they're the same one with a line break. I don't think you can if-block a partial statement

Reply:
Oh I see the line breaks now, yeah fine